MapboxDirections/MBDirections.swift

```diff
@@ -128,14 +131,27 @@ open class Directions: NSObject {
      - parameter host: An optional hostname to the server API. The [Mapbox Directions API](https://www.mapbox.com/api-documentation/?language=Swift#directions) endpoint is used by default.
      */
     public init(accessToken: String?, host: String?) {
-        let accessToken = accessToken ?? defaultAccessToken
-        assert(accessToken != nil && !accessToken!.isEmpty, "A Mapbox access token is required. Go to <https://www.mapbox.com/studio/account/tokens/>. In Info.plist, set the MGLMapboxAccessToken key to your access token, or use the Directions(accessToken:host:) initializer.")
-        
-        self.accessToken = accessToken!
-        
-        var baseURLComponents = URLComponents()
-        baseURLComponents.scheme = "https"
-        baseURLComponents.host = host ?? "api.mapbox.com"
+		var baseURLComponents: URLComponents
+        if host == nil {
+            let accessToken = accessToken ?? defaultAccessToken
+            assert(accessToken != nil && !accessToken!.isEmpty, "A Mapbox access token is required. Go to <https://www.mapbox.com/studio/account/tokens/>. In Info.plist, set the MGLMapboxAccessToken key to your access token, or use the Directions(accessToken:host:) initializer.")
+
+            self.accessToken = accessToken
+			baseURLComponents = URLComponents()
+			baseURLComponents.scheme = "https"
+			baseURLComponents.host = mapboxHost
+        }
+		else
```

Comment:
Could you fix the indentation here and also put curly braces on the same line as the else? `} else {`

Reply:
I use tabs for indentation in my Xcode... is that a problem ?
I moved the brackets correctly.

Reply:
👍 , tabs should be fine. Just trying to keep the library consistent.